tests/unit/test_contrib/test_piccolo_orm/test_piccolo_orm_dto.py

```diff
@@ -79,7 +80,8 @@ class _Table(Table):
 
 
 def test_piccolo_dto_openapi_spec_generation() -> None:
-    app = Litestar(route_handlers=[retrieve_studio, retrieve_venues, create_concert], dto=PiccoloDTO)
+    DTOBackend._seen_model_names = set()
```

Comment:
If this is needed to ensure test isolation, it should be a global fixture with `autouse=True`